### DISCUSSION TITLE
Authentication Error after a few hours

### DISCUSSION THREAD

**Discussion by Logrythmik** [directus/directus#15143] [Archive]
### Describe the Bug

After some amount of time, the requests made to pull items into the UI fail with a 403. Logging out then back in makes no difference and neither does a force-refresh. The only thing that seems to work is any change to the 'permissions' set.  Once a change is made there -- it seems to work again, but only for some time.

![image](https://user-images.githubusercontent.com/137494/184499475-d2133168-3251-4191-bc37-a1116468d46f.png)

### To Reproduce

For use, we just need to wait a few hours and suddenly it's broken again. 

### Errors Shown

As admin:
[FORBIDDEN] You don't have permission to access this.

### What version of Directus are you using?

9.15.1

### What version of Node.js are you using?

16.16.0

### What database are you using?

MySQL

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by azrikahar** [comment]
> For use, we just need to wait a few hours and suddenly it's broken again.

This may be a bit too intermittent to be reproducible reliably. Is there a failed `/auth/refresh` API call, or something along those lines when it starts to happen? Have you customized `ACCESS_TOKEN_TTL` or `REFRESH_TOKEN_TTL` for your Directus instance? 

What `CACHE_STORE` are you using? Since you mentioned making changes to permissions seems to resolve things for you, I'm wondering if it's because of faulty permissions cache due to `CACHE_PERMISSIONS`, which is enabled by default. Can you try setting `CACHE_PERMISSIONS` to false and see whether it persists for you?

**Comment by azrikahar** [comment]
Let's continue debugging in discussion first while we investigate the root cause here 👍 currently the following quote does sound like permission cache related, which would then depend on the cache config used to understand more on what could be happening here:

> The only thing that seems to work is any change to the 'permissions' set. Once a change is made there -- it seems to work again, but only for some time.
